controls/listview/cells/template-cell.md

```diff
@@ -23,7 +23,7 @@ Create a view model that will be the source of the ListView:
 
 Define the ListView control either in XAML.
 
-**Define the ListVew in XAML**
+**Define the ListView in XAML**
```

Comment:
Is this a step, a caption, or an introductory sentence? Use the appropriate style.